src/containers/ProjectContainer/ProjectContainer.js

```diff
@@ -2,30 +2,41 @@ import React from 'react';
 import ProjectCard from '../ProjectCard/ProjectCard';
 import { connect } from 'react-redux';
 
-export const ProjectContainer = (props) => {
-  const { searching, search, projects, palettes, loading, error } = props;
-  let projectCards;
+export const ProjectContainer = ({ error, searching, projects, palettes }) => {
+  const sortItems = (items) => {
+    let itemsCopy = items.slice();
+    itemsCopy.sort((a, b) => {
+      if (a.updated_at > b.updated_at) return -1;
+      if (b.updated_at < a.updated_at) return 1;
+      return 0;
+    });
+    return itemsCopy
+  }
+
+  const sortPalettes = (id) => {
```

Comment:
can we call this something else, it was a little confusing reading through this the first time, nothing is being sorted here until we pass it to the sortItems function. filterProjectPalettes maybe?